.golangci.yml

```diff
@@ -33,6 +33,9 @@ linters:
       - path: '_test\.go'
         linters:
           - err113
+      - path: 'tools/local/main.go'
+        linters:
+          - forbidigo
```

Comment:
This should ignore `use of `fmt.Printf` forbidden by pattern `^(fmt\.Print(|f|ln)|print|println)$` (forbidigo)` [here](https://github.com/Twingate/kubernetes-access-gateway/pull/98/files#diff-3ccaf05e43e37faad3b43650d29144b55251f871b017e870b8048539ced63903R135-R148) since we want to print out the run instruction

Reply:
This config file should be reserved for generic rules like across certain file types or folder. If we need to disable a file or a few lines, we should use [`nolint` directive](https://golangci-lint.run/docs/linters/false-positives/) instead.